templates/show-task-table.html

```diff
@@ -33,13 +33,33 @@
                 <tbody>
                 {% for task in tasks %}
                   <tr>
-                    <td><a href="https://en.wikipedia.org/wiki/{{ task.article_name|replace:" " }}">{{ task.article_name }}</a>
-                         {% if logged_in_user and task.status == "open" or logged_in_user and task.status == "claimed" and task.claimed_by == logged_in_user %} <i class="fa fa-pencil-square-o" onclick="taskForm({{forloop.counter0}})"></i> {% endif %}
-                    </td>
+                    <td><a href="https://en.wikipedia.org/wiki/{{ task.article_name|replace:' ' }}">{{ task.article_name }}</a></td>
                     <td>{{ task.description }}</td>
                     <td>{{ task.created_by }}</td>
-                    <td>{{ task.status }}</td>
-                    <td>{{ task.progress }}</td>
+                    <td>
+                        {% if logged_in_user and task.status == "Open" or logged_in_user and task.status == "Claimed" and task.claimed_by == logged_in_user %}
```

Comment:
We probably also want to allow the owner of the worklist to change the status, even if they have not claimed the task, right?

Also, it's probably clearer to put this computation in the handler rather than in the template. How about passing `can_modify_status` and `can_modify_progress` variables to the template from the handler?

Reply:
If the task has Open status then any can change it's status. But if the task is claimed then I guess the person who has claimed the task should only edit it as he/she will be working on it. What do you think?

I've put the computation on the server side as utility functions.

Reply:
> If the task has Open status then any can change it's status. But if the task is claimed then I guess the person who has claimed the task should only edit it as he/she will be working on it. What do you think?

The use case I have in mind is a long(ish)-lived worklist, that maybe gets used across multiple days or events. If someone claims an article on day 1, but doesn't finish it and doesn't show up on other days/events, we should have some easy way for the owner to just reset the status so someone else can claim the task.

...by the way, for the future, we might even want to allow the owner to click a button to reset all claimants, or maybe select a few tasks then click a button to reset them.

Reply:
Makes sense. I've made the changes.